src/main/java/seedu/address/model/contact/Contact.java

```diff
@@ -254,16 +260,40 @@ public Contact clearAllLinks() {
         return updatedContact;
     }
 
+    /**
+     * Marks contact object that calls this method.
+     *
+     * @return The contact that has been marked.
+     */
+    public Contact markContact() {
+        Contact updatedContact =
+            new Contact(name, phone, email, address, zoomLink, telegramHandle, tags, uuid, linkedEvents, true);
+        addToMap(updatedContact);
+        return updatedContact;
+    }
+
+    /**
+     * Removes the mark in the contact object that calls this method.
+     *
+     * @return The contact that has been un-marked.
+     */
+    public Contact unmarkContact() {
+        Contact updatedContact =
+            new Contact(name, phone, email, address, zoomLink, telegramHandle, tags, uuid, linkedEvents, false);
+        addToMap(updatedContact);
+        return updatedContact;
+    }
+
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
         builder.append(getName())
-                .append("; Email: ")
-                .append(getEmail())
-                .append(getPhone() != null ? "; Phone: " + getPhone() : "") // optional
-                .append(getAddress() != null ? "; Address: " + getAddress() : "") // optional
-                .append(getZoomLink() != null ? "; Zoom Link: " + getZoomLink() : "") // optional
-                .append(getTelegramHandle() != null ? "; Telegram: " + getTelegramHandle() : ""); // optional
+            .append("; Email: ")
+            .append(getEmail())
+            .append(getPhone() != null ? "; Phone: " + getPhone() : "") // optional
+            .append(getAddress() != null ? "; Address: " + getAddress() : "") // optional
+            .append(getZoomLink() != null ? "; Zoom Link: " + getZoomLink() : "") // optional
+            .append(getTelegramHandle() != null ? "; Telegram: " + getTelegramHandle() : ""); // optional
```

Comment:
Why are you changing all the indentations from 8 to 4 spaces? Java coding standard is 8 spaces for wrapped lines.